config/k8s/job-build.jinja2

```diff
@@ -94,7 +94,8 @@ set -x; \
 ./kci_build make_kernel && \
 ./kci_build make_modules && \
 ./kci_build make_dtbs && \
-./kci_build make_kselftest; \
+./kci_build make_kselftest && \
+./kci_build make_perftool; \
```

Comment:
We'll have to see whether we want to build perf with all the kernel builds, or if we should have an option to only select some of them.  Part of that will depend on the available build resources and the extra storage required to store the artifacts.

Reply:
I see `CONFIG_PERF_EVENTS` is enabled in `multi_v7_defconfig` but not in arm64 defconfig or x86_64_defconfig.  So the `is_enabled()` method will currently result in perf being built only for arm, unless we define a config fragment in `build-configs.yaml` to enable it.  Maybe we could have a fragment to force it to be disabled, and another one to enable it.  Then we could explicitly configure which builds have perf enabled and which ones don't.

Reply:
Sounds good. I updated this commit to add a new fragment to include a few configs (though they are selected by default defconfig).  now is_enabled() will check if this fragment is presented.
I'm not sure which arch would like to use this, so I added this fragment into default_fragments as well.

Reply:
Actually, in `init/Kconfig`:
```
config PERF_EVENTS
	bool "Kernel performance events and counters"
	default y if PROFILING
	depends on HAVE_PERF_EVENTS
```
And `CONFIG_PROFILING` is enabled in all the arch defconfigs as far as I can tell.  So we'll just need a config fragment to disable it if we want to have builds without perf.  But first, let's see if this builds fine and how much extra resources it takes as maybe we can just have perf always enabled in all the builds.

Reply:
Slight correction: `multi_v7_defconfig` doesn't have `CONFIG_PROFILING=y` but it does have `CONFIG_PERF_EVENTS`...  Same conclusion though.